### PR TITLE
Add more detail about CLI parameters that no longer supported.

### PR DESCRIPTION
**Pull request by frankfliu**
*Issue #, if available:*

*Description of changes:*

**To run CI tests on your changes refer [README.md](ci/README.md)**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -64,6 +64,15 @@ MMS doesn't support authentication natively. To avoid unauthorized access, MMS o
 * inference_address: inference API binding address, default: http://127.0.0.1:8080
 * management_address: management API binding address, default: http://127.0.0.1:8081
 
+Here is a couple of examples:
```
Comment (aaronmarkham) [inline docs/configuration.md]:
```suggestion
Here are a couple of examples:
```

File: docs/migration.md [hunk 2]
```diff
@@ -26,9 +26,19 @@ MMS 1.0 made following changes for pip installation package:
 See more detail: [Install MMS](install.md)
 
 ## Command line interface
-MMS 1.0 made some parameter changes in `mxnet-model-server` command line tool. The old command line parameters won't work any more.
-
-Please see [Command Line Interface](server.md#command-line-interface)
+MMS 1.0 made some parameter changes in `mxnet-model-server` command line tool. Following old command line parameters won't work any more:
```
Comment (aaronmarkham) [inline docs/migration.md]:
```suggestion
MMS 1.0 made some parameter changes in `mxnet-model-server` command line tool. The following command line parameters from previous versions will no longer function:
```

File: docs/migration.md [hunk 3]
```diff
@@ -26,9 +26,19 @@ MMS 1.0 made following changes for pip installation package:
...
+
+* --service, See [Register model](management_api.md#register-a-model) for how to override service entry-point.
+* --gen-api, See [API description](inference_api#api-description) for how to generate your swagger client code.
+* --port, See [Configure MMS listening port](configuration.md#configure-mms-listening-port) for how to configure port.
```
Comment (aaronmarkham) [inline docs/migration.md]:
```suggestion
* --port, See [Configure MMS listening port](configuration.md#configure-mms-listening-port) for how to configure MMS listening ports.
```

File: docs/migration.md [hunk 4]
```diff
@@ -26,9 +26,19 @@ MMS 1.0 made following changes for pip installation package:
...
+* --host, See [Configure MMS listening port](configuration.md#configure-mms-listening-port) for how to bind to specific network interface.
+* --gpu, See [Config properties](configuration.md#other-properties) for how to limit number of GPUs.
```
Comment (aaronmarkham) [inline docs/migration.md]:
```suggestion
* --gpu, See [Config properties](configuration.md#other-properties) for how to limit the number of GPUs.
```

File: docs/migration.md [hunk 5]
```diff
@@ -26,9 +26,19 @@ MMS 1.0 made following changes for pip installation package:
...
+* --log-file, See [Logging](#logging) section for how to configure logging.
```
Comment (aaronmarkham) [inline docs/migration.md]:
```suggestion
* --log-file, See [Logging](#logging) for how to specify a log file.
```

File: docs/migration.md [hunk 6]
```diff
@@ -26,9 +26,19 @@ MMS 1.0 made following changes for pip installation package:
...
+* --log-rotation-time, See [Logging](#logging) section for how to configure logging.
```
Comment (aaronmarkham) [inline docs/migration.md]:
```suggestion
* --log-rotation-time, See [Logging](#logging) for how to configure log rotation.
```

File: docs/migration.md [hunk 7]
```diff
@@ -26,9 +26,19 @@ MMS 1.0 made following changes for pip installation package:
...
+* --log-level, See [Logging](#logging) section for how to configure logging.
```
Comment (aaronmarkham) [inline docs/migration.md]:
```suggestion
* --log-level, See [Logging](#logging) for how to configure log level.
```

File: docs/migration.md [hunk 8]
```diff
@@ -26,9 +26,19 @@ MMS 1.0 made following changes for pip installation package:
...
+* --metrics-write-to, See [Metrics](#metrics) section for how to configure metrics.
```
Comment (aaronmarkham) [inline docs/migration.md]:
```suggestion
* --metrics-write-to, See [Metrics](#metrics) for how to configure metrics.
```

File: docs/migration.md [hunk 9]
```diff
@@ -26,9 +26,19 @@ MMS 1.0 made following changes for pip installation package:
...
+
+Please see [Command Line Interface](server.md#command-line-interface) for new parameters.
```
Comment (aaronmarkham) [inline docs/migration.md]:
```suggestion
For further information on the parameters' updates, please see the [Command Line Interface](server.md#command-line-interface) section of the server documentation.
```

**Comment by frankfliu** [comment]
Earlier PR has been merged already. I'm making changes based on feedback.

**Pull request merged**